cylc/flow/workflow_files.py

```diff
@@ -1491,6 +1502,51 @@ def validate_source_dir(source, flow_name):
     check_flow_file(source, logger=None)
 
 
+def get_sym_dirs(symlink_dirs: str) -> Dict[str, Dict[str, Any]]:
```

Comment:
Suggest calling this something like `parse_comma_sep_symlink_dirs` to avoid confusion with `get_symlink_dirs`